src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.MoveFile.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+public partial class FileSystemWatcher_4000_Tests
+{
+    #region WindowsTests
+
+    [Theory]
+    [InlineData(WatcherChangeTypes.Changed, "changed", false)]
```

Comment:
Yeah, these can't be parallelized in their current state because of the usage of the Utility.CreateTestDirectory() calls. I'm going to change it to use guids for the directory/file names and only have the watchers watch their owned directory and that should solve the file concurrency issues. 

I'm not familiar with xunit parallelization of Theories; does it happen automagically or do I need to specify something?